content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -23,6 +23,22 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 
 Description here.
 
+## Android Support
+
+<div class="release-feature-authors">authors: @mockersf, @slyedoc</div>
+
+<div>
+<img src="./android%20emulator.png" alt="Android emulator running Bevy" width="50%" /><img src="./ios%20emulator.png" alt="iOS emulator running Bevy" width="50%" />
```

Comment:
We might want to sort out a different android screenshot given the stretching.  Maybe better to snag a screenshot from hardware itself. Unless things have changed I don't think this is a problem on real hardware?

Regardless, that can wait until a later pass.

Reply:
I only tried on landscape mode on hardware and it wasn't an issue... but I currently don't have android hardware available

Reply:
updated the screenshots to be in landscape and they don't have stretching issue